sci-biology/aghermann/Manifest

```diff
@@ -1,3 +1,2 @@
-DIST aghermann-1.0.3.tar.xz 499492 SHA256 fa4d0543665d8e4767ff61e436ffee7249b1767e72bd4ef83e7d2a0eb03b66c0 SHA512 61b84aebfd0f22f7f831508652c8994dcd3861b354fc29022c7717b995f901200d541c491b7fd9328ccc9af7dd71779a5d8a3fc18e75fa78fc48b9ff98d72ee0 WHIRLPOOL 0f99b9d521d15fa513df905359db3f627376c83988c7b7a7bd1ad1617591acb25aa74941dc4e754443f4f879852258d65323580a2e0ae2c79024fc80c692af83
-DIST aghermann-1.0.4.tar.xz 499872 SHA256 522a8c571c9e8aaecfb997f057a01298f804bff139c052d15b3347bdf2a58722 SHA512 ff891f71480ae25cce884bfea714c11eb0536cccfe4ade8c9b742d74db63dba6664e80d7f53272e95de1e04e1fdfb895ef278106a31a1d904e95cb062c4277c8 WHIRLPOOL fbfc6c3084b6d46f460b387cba521726b6fea4f4cacfd7907011507db66cc86700c7e36a1b6823d7a7fbd04d06986ec2e12654048544730765516265799af2f9
-DIST aghermann-1.0.5.tar.xz 500112 SHA256 a0c6ccd2e740f283559b92ee6fa7ff5c92699bae72cf10a7a7f6fb53e45a258e SHA512 3355664dfa59d81c110e5da4b883c09729fee440328b6c5b568f2b412d1ad3e09503832acfe4874a7b45f3330446bb2e3e144607e3af922c6ce2484162d6d4ab WHIRLPOOL ec8f5662345a9f67d59dfe7e80b8396eefcaba45e4be8ddd899e842ed35a617d92ea503d3380868f5e851a3b90d06018ecae6ad986bf015aee4f8aa14c89a211
+DIST aghermann-1.0.7.tar.xz 509336 SHA256 892b7cbb4a19554a772779e861a9f01b1bf753605722afb70a6ab362bebb2751 SHA512 bec591865a305ef3888c2f2986c3417139a853a9d5b95894c7b512fe48a220e40199a2c4ba84fa79c960e666e33b1f3371fbdc46c163d699a2041d0ba67eaebd WHIRLPOOL bcd02dcf06ac91f62b61014f6dace95d7721f0ba5e95e4080547e9c0a8b1a461fe3691da4360c8c33ab7b4ba04a35bcb6a0f006fcb4f9df5f918403ee5587627
+EBUILD aghermann-1.0.7.ebuild 667 SHA256 5ce4e22d937751368f976347b0313f0807c4e26f5b5713456df506bdcb51d5d7 SHA512 ab7d88839d2da8b32b903225f37898c1b0232fa74739823fe1c84e1a2fbf72b445de6627bd1df8dddb9ff3b2f46de3251037e23d4a78476bf274770382d5ec29 WHIRLPOOL 2434980c9fc21185b15e40986ead581007e372fc895f122ba50b4e7a720ddf9063d0706df250691cc165d14bdfe0bee2bd6e0ff85f32ad542070e3cdab695b65
```

Comment:
It shouldn't be necessary to add an Manifest entry for the EBUILD, as the science overlay uses thin manifests. Don't know what went wrong here.